src/components/EditableArea.tsx

```diff
@@ -53,7 +55,7 @@ export const EditableArea = <
   if (isEditor) {
     return (
       <Comment
-        openComment={templateAnnotations?.[content['@path']]}
+        openComment={templateAnnotations?.[originalContent?.['@path']]}
```

Comment:
the variant content has a different node path which will not have a template annotation for it hence the use of the original content node path here